### PR TITLE
[Keyboard]Remove RGB light implementation of ID67, revert the file structure changes by #15558

### PR DESCRIPTION
**Pull request by peepeetee**
## Description
After a lengthy discussion with everyone involved in #15558 (@thewerther @vinorodrigues), and talking to IDOBAO, we have decided to remove the RGB light implementation kept in for legacy reasons, and leave RGB matrix as the only implementation for ID67.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/id67/readme.md [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
-Make examples for this keyboard (after setting up your build environment):
 
-    make id67/default_rgb:default
+## Compiling and flashing
 
-    make id67/rgb:default
+Make example for this keyboard (after setting up your build environment):
 
-Flashing examples for this keyboard:
+    make id67/rgb:default
```
Comment (peepeetee) [inline keyboards/id67/readme.md:22]:
```suggestion
    make id67:default
```

File: keyboards/id67/readme.md [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
 
-    make id67/default_rgb:default:flash
+Flashing example for this keyboard:
 
     make id67/rgb:default:flash
```
Comment (peepeetee) [inline keyboards/id67/readme.md:26]:
```suggestion
    make id67:default:flash
```

File: keyboards/id67/keymaps/via/config.h [hunk 3]
```diff
@@ -14,4 +14,6 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "default_rgb.h"
+#pragma once
+
+#define VIA_QMK_RGBLIGHT_ENABLE
```
Comment (thewerther) [inline keyboards/id67/keymaps/via/config.h:19]:
```
#if defined(RGB_MATRIX_ENABLE)
    #define VIA_QMK_RGBLIGHT_ENABLE
#endif
```

File: keyboards/id67/id67.c [hunk 4]
```diff
@@ -43,7 +43,7 @@ led_config_t g_led_config = { {
     1, 4, 1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,    // second row
     1, 1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,    // third row
```
Comment (peepeetee) [inline keyboards/id67/id67.c:44]:
```suggestion
    1, 1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 9,    // third row
```

**Review by filterpaper — CHANGES_REQUESTED** [review]

File: keyboards/id67/keymaps/via/config.h [hunk 5]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2021 Tybera
+/* Copyright 2021 thewerther
```
Comment (filterpaper) [inline keyboards/id67/keymaps/via/config.h:1]:
```suggestion
/* Copyright 2021 Tybera
 * Copyright 2022 thewerther
```

It is probably good to retain name(s) of previous contributors in the source.

**Review by filterpaper — CHANGES_REQUESTED** [review]

File: keyboards/id67/keymaps/via/config.h (hunk 5 above)
Comment (peepeetee) [reply keyboards/id67/keymaps/via/config.h:1]: This diff is a side effect of moving the files in and out of a folder. But, the advice is still valid

**Comment by thewerther** [comment]
Hi, can we get this merged? Or is there still something missing?

**Review by drashna — APPROVED** [review]

**Review by thewerther — APPROVED** [review]

**Comment by vinorodrigues** [comment]
Since #15662 has moved the files to the `idobao` folder - this branch + PR should also have it's files moved to facilitate merger.

**Comment by peepeetee** [comment]
> Since #15662 (already merged) has moved the files to the `idobao` folder - this branch + PR should also have it's files moved to facilitate merger.

Done

**Review by vinorodrigues — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**